waiter/src/waiter/shell_scheduler.clj

```diff
@@ -97,7 +97,7 @@
   (try
     (log/info "killing process:" instance)
     (.destroyForcibly process)
-    (sh/sh "pkill" "-9" "-s" (str pid))
+    (sh/sh "pkill" "-9" "-g" (str pid))
```

Comment:
How do you feel about launching processes in unit tests? Otherwise I think we just rely on the integration tests not blowing up their container.